recipe-server/normandy/recipes/tests/test_models.py

```diff
@@ -283,6 +283,12 @@ def test_recipe_update_locales(self):
         recipe.update(locales=[])
         assert recipe.locales.count() == 0
 
+    def test_recipe_update_arguments(self):
```

Comment:
Can you also add an API test that reproduces the error POSTing to the view? That'd be closer to the situation we're actually seeing the error in.